src/Halogen/HTML/Events/Types.purs

```diff
@@ -33,6 +36,23 @@ type MouseEvent =
   , which :: Number
   )
 
+-- | Identifies the additional fields which are available on drag events.
+type DragEvent =
+  ( button :: Number
+  , detail :: Number
+  , relatedTarget :: HTMLElement
+  , clientX :: Number
+  , clientY :: Number
+  , screenX	:: Number
+  , screenY	:: Number
+  , ctrlKey	:: Boolean
+  , shiftKey :: Boolean
+  , altKey :: Boolean
+  , metaKey	:: Boolean
+  , which :: Number
+  , dataTransfer :: Nullable DataTransfer
+  )
```

Comment:
`DragEvent` inherits from `MouseEvent`. I've just copied the fields since it's the most straightforward thing to do. We probably could make it nicer, e.g. by adding `fields` parameter to `MouseEvent` and other events.